idb/postgres/internal/ledger_for_evaluator/ledger_for_evaluator_test.go

```diff
@@ -2,13 +2,14 @@ package ledgerforevaluator_test
 
 import (
 	"context"
-	"database/sql"
 	"testing"
 
 	"github.com/algorand/go-algorand/crypto"
 	"github.com/algorand/go-algorand/data/basics"
 	"github.com/algorand/go-algorand/data/bookkeeping"
 	"github.com/algorand/go-algorand/data/transactions"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
```

Comment:
this should be grouped with the other non-algorand 3rd party packages

Reply:
I removed the separation between the two, since we already have lots of code where those are together.